web/views.py

```diff
@@ -444,7 +442,7 @@ class GamesFeed(Feed):
     link = "/games/"
     decription = "Feed for every game of Academy started."
 
-    def items(self) -> Iterable[Game]:
+    def items(self):
```

Comment:
Why was the return type removed?

Reply:
It was mistakenly added to the commit. They should be added back.

Some type annotations do not work with older versions of python.